### ISSUE TITLE
OSError: read returned empty

### ISSUE THREAD

**Issue by Net-Felix** [nicoboss/nsz#60]
```

             NSZ v3.1   ,;:;;,
                       ;;;;;
               .=',    ;:;;:,
              /_', "=. ';:;:;
              @=:__,  \,;:;:'
                _(\.=  ;:;;'
               `"_(  _/="`
                `"'

Solid compressing (level 18) D:\downloads\switch\01006F8002326000.nsp -> D:\SWITCHTOOLS\nsz\01006F8002326000.nsz
[ADDING]     83d23343779ecd95e7355c39e931e24a.cnmt.nca 3584 bytes to NSP
not packed!
[ADDING]     80cbc793d37b6f493ba53561106bb5c6.nca 6692945920 bytes to NSP
Traceback (most recent call last):
  File "nsz\SolidCompressor.py", line 128, in solidCompressNsp
  File "nsz\SolidCompressor.py", line 114, in processContainer
  File "nsz\Fs\File.py", line 331, in read
OSError: read returned empty 0xffee5010

Done!
```

then the console hangs at:
`Compressing   0%|                                                                    |   0/100 MiB [00:07<?, 0.00 MiB/s]`

**Comment by nicoboss** [comment]
Thanks a lot for reporting that. I was able to reproduce it and will fix it as soon.

**Comment by nicoboss** [comment]
This issue should be fixed in the latest [NSZ v4.1.0](https://github.com/nicoboss/nsz/releases/tag/4.2.0) release. Please let me know if you still experience this issue on the latest release.

I have no idea what exactly caused and fixed this issue but I can confirm that I can no longer reproduce it on the latest version of NSZ. If someone in the future stumbles upon an "OSError: read returned empty" issue please use --verify first to confirm it's not a bad dump (which seems unlikely in this case as I was able to reproduce it in the past but others reported getting "OSError: read returned empty" on bad dumps).
